### PR TITLE
Remove EAP references

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) removes references to the EAP, updates links to support page, and rearranges some pages.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/product-recs.html

### CODE REVIEW

File: src/recommendations/install-configure.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+If needed, you can [uninstall the product-recommendations module]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/install/cli/install-cli-uninstall-mods.html).
+
+## Configure Product Recommendations {#configure}
+
+After you install the `magento/product-recommendations` module, you must link the module to Magento's SaaS Services by [retrieving the API Key and specifying the SaaS Environment](https://docs.magento.com/m2/ce/user_guide/configuration/services/saas.html).
+
+When you add the API key to Magento Services, ensure that the [cron]({{ site.baseurl }}/guides/v{{ site.version }}/config-guide/cli/config-cli-subcommands-cron.html) jobs and the [indexers]({{ site.baseurl }}/guides/v{{ site.version }}/config-guide/cli/config-cli-subcommands-index.html) are running and the `Product Feed` indexer is set to `Update by Schedule`.
+
+When you successfully link to Magento Services through the API key and specify the SaaS Environment, the catalog sync initiates and behavioral data collection begins on your storefront.
```
Comment (misha-kotov) [inline src/recommendations/install-configure.md:48]: we should link to "verify events" topic here

File: src/recommendations/install-configure.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Uninstall Product Recommendations
+
+If needed, you can [uninstall the product-recommendations module]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/install/cli/install-cli-uninstall-mods.html).
+
+## Configure Product Recommendations {#configure}
+
+After you install the `magento/product-recommendations` module, you must link the module to Magento's SaaS Services by [retrieving the API Key and specifying the SaaS Environment](https://docs.magento.com/m2/ce/user_guide/configuration/services/saas.html).
+
+When you add the API key to Magento Services, ensure that the [cron]({{ site.baseurl }}/guides/v{{ site.version }}/config-guide/cli/config-cli-subcommands-cron.html) jobs and the [indexers]({{ site.baseurl }}/guides/v{{ site.version }}/config-guide/cli/config-cli-subcommands-index.html) are running and the `Product Feed` indexer is set to `Update by Schedule`.
```
Comment (misha-kotov) [inline src/recommendations/install-configure.md:46]: I would specify which part component this is pertinent to: 

> To ensure catalog export is running correctly, confirm that the cron jobs and the indexers....

File: src/recommendations/install-configure.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+composer update magento/product-recommendations --with-dependencies
+```
+
+### Uninstall Product Recommendations
+
+If needed, you can [uninstall the product-recommendations module]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/install/cli/install-cli-uninstall-mods.html).
+
+## Configure Product Recommendations {#configure}
+
+After you install the `magento/product-recommendations` module, you must link the module to Magento's SaaS Services by [retrieving the API Key and specifying the SaaS Environment](https://docs.magento.com/m2/ce/user_guide/configuration/services/saas.html).
```
Comment (misha-kotov) [inline src/recommendations/install-configure.md:44]: can we simplify by saying

> you must configure the module by specifying the API Key and selecting a SaaS Environment

I think I'd avoid using the "linking" verbiage here (it's fine in the userguide intro)

File: src/recommendations/install-configure.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+   -  **module-data-services** — This module enables behavioral data collection by tracking [user events on the page]({{ page.baseurl }}/recommendations/events.html). This type of data is required by Adobe Sensei to compute product affinities based on production shopper behavior like product views, products added to a cart, and checkouts. Adobe Sensei then uses this information to create and train machine learning models for each website and storeview. This unlocks recommendation types like "Customers who viewed this, also viewed...", which automatically adjusts with shopper behavior over time. Magento and Adobe Sensei do not collect personally identifiable information.
+
+   -  **saas-export** — This module syncs catalog data. This type of data provides product information to the Product Recommendations service so it can accurately return product names, pricing, images, URLs, inventory and availability, and other attributes.
+
+       {:.bs-callout-info}
+       If you prefer, you can install the above modules explicitly using Composer: `composer require magento/module-data-services` and `composer require magento/saas-export`
+
+   -  Other modules responsible for configuration
```
[the remark this answers is not in the document]
Comment (erikmarr) [reply src/recommendations/install-configure.md:28]: removed

**Comment by erikmarr** [comment]
running tests

**Pull request merged**